exporter/collector/metricsexporter.go

```diff
@@ -430,7 +431,7 @@ func (m *metricMapper) summaryPointToTimeSeries(
 	for i := 0; i < quantiles.Len(); i++ {
 		quantile := quantiles.At(i)
 		pLabel := labels{
-			"quantile": fmt.Sprintf("%f", quantile.Quantile()),
+			"quantile": strconv.FormatFloat(quantile.Quantile(), 'f', -1, 64),
```

Comment:
Are there any conventions from Prometheus land that we should make sure we're following here? (e.g. should 0 be reported as `0` or `0.0` or something else? Ditto for 1) I think as-is this will report 0 as `0` and 1 as `1`.

Reply:
It reports them a `0` and `1`.  See these test-cases: https://github.com/prometheus/prometheus/blob/4c56a193c518ae6f56008b0a4c850a9c3f1477c6/model/textparse/promtestdata.txt#L3

Reply:
So I think we are good on that front